botbuilder-instumentation/HttpCommunication.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using BotBuilder.Instrumentation.Interfaces;
+
+namespace BotBuilder.Instrumentation
+{
+    public class HttpCommunication : IHttpCommunication
+    {
+        private const string JsonMimeType = "application/json";
+
+        public async Task<string> SendAsync(string baseEndpoint, string route, IDictionary<string, string> headers, byte[] data)
```

Comment:
The name `SendAsync` is not informative regarding the HTTP actions used to send the data.
I would rename it to PostAsync because that's what it does.
It will allow us to add in the future GetAsync.